src/files/rsql-filter-expression.ts

```diff
@@ -14,7 +16,9 @@ export class RSQLFilterExpression {
     value: string | Array<string | number | boolean> | Date | number | boolean | undefined
   ) {
     this.field = field;
-    if (Object.values(Operators).includes(operator)) {
+    let object: Array<Object> = Object.keys(Operators).map(test => (Operators as any)[test]);
+
+    if (includes(object, operator)) {
```

Comment:
I'd rather not bring in a library if we don't have to.  I have another proposal here:
```
if (typeof operator === 'object' && this.instanceOfCustomOperator(operator)) {
      this.operator = undefined;
      this.customOperator = operator as CustomOperator;
    } else {
      this.operator = operator as Operators;
      this.customOperator = undefined;
    }
```

and then at the end of the class:
```
  private instanceOfCustomOperator(object: any): object is CustomOperator {
    return 'convertToRSQLString' in object;
  }
```

I think this should work in IE11 as well, but I haven't tested it.  This should hopefully be a less expensive operation than what I had originally and what is being proposed here.

Reply:
I've changed it to the suggested code. I also tested it locally on IE and it looks like works as well.